electron/utils/write-metadata.ts

```
@@ -0,0 +1,22 @@
+import { ExifTool } from "exiftool-vendored";
+import logit from "../utils/logit";
+
+const writeMetadata = async (
+  outPath: string,
+  metadata: Record<string, any>,
+  exiftool: ExifTool,
+) => {
+  logit("🖊️ Writing Metadata to path: ", outPath);
+  const metadataCopy = { ...metadata };
+
+  if (metadataCopy.hasOwnProperty("FileName")) delete metadataCopy.FileName;
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)